### PR TITLE
T1.9 test case

### PR DESCRIPTION
**Pull request by kevinwish**

### CODE REVIEW

File: public/electron.js [hunk 1]
```diff
@@ -70,4 +70,4 @@ app.on('activate', function() {
 });
 
 // In this file you can include the rest of your app's specific main process
-// code. You can also put them in separate files and require them here.
\ No newline at end of file
+// code. You can also put them in separate files and require them here.
```
Comment (braunsonm) [inline public/electron.js]: Revert this change

File: src/__tests__/ParserComponent.test.tsx [hunk 2]
```diff
@@ -265,6 +265,60 @@ describe('Csv FileEvents processed correctly\n', () => {
...
+    onChange={onChangeMock}
+  />);
+
+  // clear the on change event mock and the enzyme component
+  beforeEach(() => {
+    onChangeMock.mockClear();
+    comp.setState({data: []});
+  });
+
+  it('file name with \\', async () => {
```
Comment (braunsonm) [inline src/__tests__/ParserComponent.test.tsx]: Osgood was wanting some more docs about what this is really testing and what the expected output would be

File: src/__tests__/ParserComponent.test.tsx [hunk 3]
```diff
@@ -265,6 +265,60 @@ describe('Csv FileEvents processed correctly\n', () => {
...
+        {type: '.csv,text/csv'},
+    );
+    const fileEvent = {target: {files: [testfile]}};
+    await comp.instance().parse(fileEvent);
+    expect(comp.state('data').length).toEqual(3);
+    expect(onChangeMock).toHaveBeenCalledTimes(1);
+  });
+
+
+  it('file name with emoji that use unicode', async () => {
```
Comment (braunsonm) [inline src/__tests__/ParserComponent.test.tsx]: Same little doc thing

File: src/__tests__/ParserComponent.test.tsx [hunk 4]
```diff
@@ -265,6 +265,60 @@ describe('Csv FileEvents processed correctly\n', () => {
...
+        '06-01-1994,5,efg\n' +
+        'November 5 1997,1,hij\n' +
+        ''],
+        'test\\.csv',
+        {type: '.csv,text/csv'},
+    );
+    const fileEvent = {target: {files: [testfile]}};
+    await comp.instance().parse(fileEvent);
+    expect(comp.state('data').length).toEqual(3);
+    expect(onChangeMock).toHaveBeenCalledTimes(1);
```
Comment (eileenvanheerde) [inline src/__tests__/ParserComponent.test.tsx]: Because you're checking this at the end of both tests, you can put this in afterEach()
